### PR TITLE
fix flexible races gnome

### PR DESCRIPTION
**Pull request by ThyWoof**

### CODE REVIEW

File: SolastaCommunityExpansion/Models/FlexibleRacesContext.cs [hunk 1]
```diff
@@ -17,15 +17,15 @@ internal static class FlexibleRacesContext
             "FlexibleRaces/&PointPoolAbilityScore4Description",
             "FlexibleRaces/&PointPoolAbilityScore4Title");
 
-        private static readonly FeatureUnlockByLevel attributeChoiceFour = new FeatureUnlockByLevel(new FeatureDefinitionPointPoolBuilder("PointPoolAbilityScore4",
+        private static readonly FeatureUnlockByLevel _ = new FeatureUnlockByLevel(new FeatureDefinitionPointPoolBuilder("PointPoolAbilityScore4",
```
Comment (PhilPJL) [inline SolastaCommunityExpansion/Models/FlexibleRacesContext.cs:20]: Does this need to be kept for backward compatibility?  If so add comment.

File: SolastaCommunityExpansion/Models/FlexibleRacesContext.cs [hunk 2]
```diff
@@ -47,7 +47,7 @@ internal static class FlexibleRacesContext
             { "HalfOrc", new List<string> { "FeatureSetHalfOrcAbilityScoreIncrease" } },
             // unofficial races
             { "FirbolgRace", new List<string> { "AttributeModifierFirbolgStrengthAbilityScoreIncrease", "AttributeModifierFirbolgWisdomAbilityScoreIncrease" } },
-            { "Gnome", new List<string> { "AttributeModifierGnomeAbilityScoreIncrease", "AttributeModifierForestGnomeAbilityScoreIncrease" } }
+            { "GnomeRace", new List<string> { "AttributeModifierGnomeAbilityScoreIncrease", "AttributeModifierForestGnomeAbilityScoreIncrease" } }
         };
 
         private static void RemoveMatchingFeature(List<FeatureUnlockByLevel> unlocks, FeatureDefinition toRemove)
```
Comment (PhilPJL) [inline SolastaCommunityExpansion/Models/FlexibleRacesContext.cs:53]: Although not part of this fix, this method would probably be better as:

```
        private static void RemoveMatchingFeature(List<FeatureUnlockByLevel> unlocks, FeatureDefinition toRemove)
        {
            unlocks.RemoveAll(f => f.FeatureDefinition.GUID == toRemove.GUID);
        }

```

File: SolastaCommunityExpansion/Models/FlexibleRacesContext.cs (hunk 1 above)
Comment (ThyWoof) [reply SolastaCommunityExpansion/Models/FlexibleRacesContext.cs]: rolled back

**Comment by ThyWoof** [comment]
half-elf is indeed 4...

**Review by PhilPJL — APPROVED** [review]

**Pull request merged**